sepes.sln

```diff
@@ -0,0 +1,48 @@
+﻿
```

Comment:
This file should be called Sepes.RestApi.sln.
The changes we have made will totally invalidate this file.
Do not try to edit use tools to fix or generate new one from scratch.